shell/platform/embedder/embedder.h

```diff
@@ -2119,6 +2121,7 @@ typedef struct {
   FlutterEngineTraceEventDurationBeginFnPtr TraceEventDurationBegin;
   FlutterEngineTraceEventDurationEndFnPtr TraceEventDurationEnd;
   FlutterEngineTraceEventInstantFnPtr TraceEventInstant;
+  FlutterEnginePostPlatformThreadTaskFnPtr PostPlatformThreadTask;
```

Comment:
New entries in embedder.h structs must always go at the end for binary compatibility reasons.